python/tests/test_writer.py

```diff
@@ -1251,3 +1251,24 @@ def test_with_deltalake_schema(tmp_path: pathlib.Path, sample_data: pa.Table):
     )
     delta_table = DeltaTable(tmp_path)
     assert delta_table.schema().to_pyarrow() == sample_data.schema
+
+
+def test_write_stats_empty_rowgroups_2169(tmp_path: pathlib.Path):
+    data = pa.table(
```

Comment:
Let's add a link here to the issue, since it might not be obvious what the number in the function name is.
```suggestion
def test_write_stats_empty_rowgroups(tmp_path: pathlib.Path):
    # https://github.com/delta-io/delta-rs/issues/2169
    data = pa.table(
```

Reply:
Good point :) @wjones127 can you approve again?